bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java

```diff
@@ -983,4 +983,29 @@ public static String escapeHtml(@Nullable String text) {
         }
         return grouped;
     }
+    
+    /**
+     * Returns lower case of ASCII characters in the string.
+     * 
+     * @param str input string
+     * @return lower case ASCII string
+     */
+    @Nullable
+    public static String toLowerCaseASCII(@Nullable String str) {
+        if (isEmptyTrimmed(str)) {
```

Comment:
This condition should be negated or used as a fast "bail-out" path. Please run tests after making such changes

Reply:
I did notice and switched it. Thanks!